nav2_world_model/src/world_model.cpp

```diff
@@ -12,67 +12,124 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <iostream>
-#include <memory>
-#include <vector>
-#include <string>
 #include "nav2_world_model/world_model.hpp"
 
-using std::vector;
-using std::string;
+#include <iostream>
+#include <memory>
 
 namespace nav2_world_model
 {
 
-WorldModel::WorldModel(rclcpp::executor::Executor & executor, const string & name)
-: Node(name),
-  tfBuffer_(get_clock()),
-  tfListener_(tfBuffer_)
+WorldModel::WorldModel()
+: nav2_lifecycle::LifecycleNode("world_model")
+{
+  RCLCPP_INFO(get_logger(), "Creating");
+
+  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>("world_model_global_costmap");
```

Comment:
I assume (now looking at #664) that the `costmap_2d_ros` creates its own tf_buffer for transforms

Reply:
@bpwilcox Yes, that's correct.